com.unity.render-pipelines.universal/Shaders/PostProcessing/UberPost.shader

```diff
@@ -68,7 +71,12 @@ Shader "Hidden/Universal Render Pipeline/UberPost"
         #define LensDirtIntensity       _LensDirt_Intensity.x
 
         #define VignetteColor           _Vignette_Params1.xyz
+    #ifdef USING_STEREO_MATRICES
+        #define VignetteCenterEye0      _Vignette_Params2.xy
```

Comment:
Since we're already differentiating with center and Eye0, Eye1 we could just pack all the XR Eye data to Params3 (or ParamsXR). I think it would be a tiny bit cleaner as it would avoid this interleaving
`params2( standard.xy, XR0.xy), params3( XR1.xy,  --/standard.xy? )` 
pattern, and be 
`params2( standard.xy, --/standard.xy? ), params3( XR0.xy, XR1.xy ) or paramsXR( XR0.xy, XR1.xy )`
as it doesn't matter where the 2 extra floats are.

Reply:
Sure, I'll get the XR single-pass data to its own variable then to group it all together